### PR TITLE
Remove deprecated features on v9

### PR DESCRIPTION
**Pull request by FabienPapet**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Remove deprecated features on v9
| Type?             | improvement
| Category?         | CO
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | -
| Related PRs       | -
| How to test?      | I only removed code deprecated in previous version and that was not used, so not behavior should be modified. I believe automated tests should tell us if something was broken although in theory it should not happen.
| Possible impacts? | No, only deprecated and not used code has been removed
| Sponsor company   |

### CODE REVIEW

**Comment by nicosomb** [comment]
Same comment as https://github.com/PrestaShop/PrestaShop/pull/31029#issuecomment-1404658645

File: classes/Tools.php [hunk 1]
```diff
@@ -2225,13 +2106,6 @@ public static function parserSQL($sql)
         return false;
     }
 
-    public static function replaceByAbsoluteURL($matches)
```
Comment (lartist) [inline classes/Tools.php:2228]: This one is not marked as deprecated

Comment (FabienPapet) [reply classes/Tools.php:2228]: Inside the method there's `        Tools::displayAsDeprecated('Use Media::replaceByAbsoluteURL($matches) instead');
`

Comment (lartist) [reply classes/Tools.php:2228]: Didn't see 👍

**Review by lartist — DISMISSED** [review]

**Review by matks — DISMISSED** [review]
Dear QA team as Fabien said, this PR only removes code deprecated in previous version and that was not used, so not behavior should be modified. Automated tests should be enough I think.

**Review by kpodemski — COMMENTED** [review]
Link list widget requires some changes:

https://github.com/PrestaShop/ps_linklist/blob/dev/src/Presenter/LinkBlockPresenter.php#L223

**Comment by paulnoelcholot** [comment]
@kpodemski @FabienPapet 
Pending the required changes, I am withdrawing the Waiting for QA label. Please add it again when the PR is ready to be tested.

**Comment by kpodemski** [comment]
@paulnoelcholot, it is ready to test, I didn't put "Request changes", it's just a comment for @FabienPapet to remember about the Link list widget + auto tests should be enough there, right?

**Review by florine2623 — COMMENTED** [review]
Hello @FabienPapet ,

Thanks to the [automated tests](https://github.com/florine2623/testing_pr/actions/runs/4132763879) and @nesrineabdmouleh , I have found these 3 issues :

In BO > Shop Parameters > Order Settings > Tab Statuses :
Screenshot 2023-02-09 at 15 18 45

In BO > Customer Service > Merchandise returns (Enable + make a return + disable) :
Screenshot 2023-02-09 at 15 21 43


In FO > Language selector is broken + Fatal error is displayed :

https://user-images.githubusercontent.com/16019289/217838375-596cd66a-d196-4085-9ce2-980b08423c21.mov

Screenshot 2023-02-09 at 15 20 12

Could you check please ?

Thanks!

**Comment by FabienPapet** [comment]
Hey @florine2623 

Thank your for testing 

- The first failing test is due to a deprecated feature that was still used, so I undeleted this.
- The second one comes from a module. The module has been updated.

So this PR is ready to test again :) 
I've relaunched the automated tests : https://github.com/FabienPapet/ga.tests.ui.pr/actions/runs/4193577871

**Review by mflasquin — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]

**Review by mflasquin — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Thanks @FabienPapet ,

I've rechecked the previous bugs and they are fixed !

It is QA ✅ !
Thanks!

**Pull request merged**